### DISCUSSION TITLE
Switch to Firefox ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [radubirsan/FlashBrowser#11] [Ideas]
Hi, first of all great job on this fork !

What about using **Firefox** (free and open-source software) instead of **chrome** (Proprietary freeware, based on open source components.) !?

**Comment by Alex313031** [comment]
@BobSquarePants It's based on Chromium, not chrome. Chromium is free and open source just like firefox.
Matter of fact, I make my own Chromium fork for linux, windows, and macos, as well as ChromiumOS (the open source counterpart to ChromeOS i.e. the OS used on Chromebooks.)
https://github.com/Alex313031/Thorium
https://github.com/Alex313031/Thorium-Win
https://github.com/Alex313031/ChromiumOS

**Comment by radubirsan** [comment]
> Hi, first of all great job on this fork !
> 
> What about using **Firefox** (free and open-source software) instead of **chrome** (Proprietary freeware, based on open source components.) !?

Yes Firefox should get more attention since it is the only alternative, it's death would mean that competition would happen only inside chromium forks.  Since this is a small hobby project, there are no plans to switch for now.
